src/cpu/gemm/gemm.cpp

```diff
@@ -192,6 +196,25 @@ dnnl_status_t gemm_s8x8s32(const char *transa, const char *transb,
                 B, LDB, bo, beta, C, LDC, co, false);
 #endif
 
+#ifdef DNNL_PPC64
+#ifdef __MMA__
+    if (!(utils::one_of(*transa, 'n', 'N', 't', 'T')
+                && utils::one_of(*transb, 'n', 'N', 't', 'T')))
+        return dnnl_unimplemented;
```

Comment:
This part is likely belong to `check_gemm_x8x8x32_input(...)` function call. Could you move it there, please?

Reply:
Moved as requested.